test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoUpdateSuite.java

```diff
@@ -246,6 +188,64 @@ private HapiApiSpec updateFailsWithInvalidMaxAutoAssociations() {
 				);
 	}
 
+	private HapiApiSpec usdFeeAsExpected() {
+		double autoAssocSlotPrice = 0.0018;
+		double baseFee = 0.00022;
+		double plusOneSlotFee = baseFee + autoAssocSlotPrice;
+		double plusTenSlotsFee = baseFee + 10 * autoAssocSlotPrice;
+
+		final var baseTxn = "baseTxn";
+		final var plusOneTxn = "plusOneTxn";
+		final var plusTenTxn = "plusTenTxn";
+
+		AtomicLong expiration = new AtomicLong();
+		return defaultHapiSpec("UsdFeeAsExpectedCryptoUpdate")
+				.given(
+						newKeyNamed("key").shape(SIMPLE),
+						cryptoCreate("payer")
+								.key("key")
+								.balance(1_000 * ONE_HBAR),
+						cryptoCreate("canonicalAccount")
+								.key("key")
+								.balance(100 * ONE_HBAR)
+								.autoRenewSecs(THREE_MONTHS_IN_SECONDS)
+								.blankMemo()
+								.payingWith("payer"),
+						cryptoCreate("autoAssocTarget")
+								.key("key")
+								.balance(100 * ONE_HBAR)
+								.autoRenewSecs(THREE_MONTHS_IN_SECONDS)
+								.blankMemo()
+								.payingWith("payer"),
+						getAccountInfo("canonicalAccount")
+								.exposingExpiry(expiration::set)
+				)
+				.when(
+						sourcing(() ->
+								cryptoUpdate("canonicalAccount")
+										.payingWith("canonicalAccount")
+										.expiring(expiration.get() + THREE_MONTHS_IN_SECONDS)
+										.blankMemo()
+										.via(baseTxn)
+						),
+						cryptoUpdate("autoAssocTarget")
+								.payingWith("autoAssocTarget")
+								.blankMemo()
+								.maxAutomaticAssociations(1)
+								.via(plusOneTxn),
+						cryptoUpdate("autoAssocTarget")
+								.payingWith("autoAssocTarget")
+								.blankMemo()
+								.maxAutomaticAssociations(11)
+								.via(plusTenTxn)
+				)
+				.then(
+						validateChargedUsd(baseTxn, baseFee),
+						validateChargedUsd(plusOneTxn, plusOneSlotFee),
+						validateChargedUsd(plusTenTxn, plusTenSlotsFee)
+				);
+	}
+
```

Comment:
nit: It seems you have simply moved this test to the bottom of the file? If that's the case, can we revert it to the place it initially was?

Reply:
This one breaks the "UsdFeeAsExpectedCryptoUpdate" test case, because of app properties overriding and simultaneous execution. It was intended to be extracted and run separately, but it needs further discussion, that is why the change was reverted.